workflow/Snakefile

```diff
@@ -40,11 +40,11 @@ BIOSAMPLE_ACTIVITES = {
 	biosample: activity for _, (biosample, activity) in 
 	BIOSAMPLE_DF[["biosample", "default_accessibility_feature"]].iterrows()
 }
+ABC_dirs = {biosample: os.path.join(RESULTS_DIR, biosample) for biosample in BIOSAMPLE_DF.biosamples}
```

Comment:
Let's update this Snakefile based on the changes you and I have made on the `main` branch